lightwood/encoders/time_series/rnn.py

```diff
@@ -12,14 +12,14 @@
 
 class RnnEncoder(BaseEncoder):
 
-    def __init__(self, encoded_vector_size=4, train_iters=75000, stop_on_error=0.8, learning_rate=0.01,
-                 is_target=False, ts_n_dims=1, max_timesteps=64):
+    def __init__(self, encoded_vector_size=4, train_iters=100, stop_on_error=0.8, learning_rate=0.01,
```

Comment:
Unrelated to the PR, but, is `stop_on_error` supposed to be this high ? Shouldn't it be closer to ~ 0.08 or thereabouts ?

Ideally we'd want to add "stop when not improving" logic to this encoder anyway (see Gym for how that's done). But for now a quick fix might involve lowering this number together with lowering `train_iters`.

Reply:
You are right, changing this to 0.01 as a quick fix.